app/views/metrics/show.html.erb

```diff
@@ -108,43 +108,54 @@
   <div class="govuk-grid-column-full section-performance">
     <h2 class="section-performance__header"><%= t ".section_headings.performance" %></h2>
 
-    <%= render 'metric_section', 
+    <%= render 'metric_section',
         metric_name: 'upviews',
-        total: @performance_data.total_upviews, 
+        total: @performance_data.total_upviews,
         short_context: nil,
         external_link: nil %>
 
     <hr class="govuk-section-break govuk-section-break--l govuk-section-break--visible">
 
-    <%= render 'metric_section', 
-        metric_name: 'pviews', 
-        total: @performance_data.total_pviews, 
+    <%= render 'metric_section',
+        metric_name: 'pviews',
+        total: @performance_data.total_pviews,
```

Comment:
I'm going to try and split the other changes out from the functional changes to make this a bit easier to read.

Reply:
Done now in 35430a156ce5437d87dc20da72513ea222c4ffdf